packages/debugger/lib/trace/sagas/index.js

```diff
@@ -25,51 +24,36 @@ export function* addSubmoduleToCount(increment = 1) {
 }
 
 export function* advance() {
-  yield put(actions.next());
-
-  debug("TOCK to take");
-  yield take([actions.TOCK, actions.END_OF_TRACE]);
-  debug("TOCK taken");
-}
-
-function* next() {
   let remaining = yield select(trace.stepsRemaining);
   debug("remaining: %o", remaining);
   let steps = yield select(trace.steps);
   debug("total steps: %o", steps.length);
   let waitingForSubmodules = 0;
 
   if (remaining > 0) {
-    debug("putting TICK");
     // updates state for current step
     waitingForSubmodules = yield select(trace.application.submoduleCount);
     yield put(actions.tick());
-    debug("put TICK");
 
     //wait for all backticks before continuing
     while (waitingForSubmodules > 0) {
-      yield take(actions.BACKTICK);
-      debug("got BACKTICK");
+      yield take(actions.TOCK);
```

Comment:
Does it make sense to keep the debug logs for tracing `actions.tick()`, `actions.TOCK()` and `actions.endTrace()`?

Reply:
Not really to my mind; I removed them because this part of the code has been unproblematic for quite a while now, and these debugs are quite noisy if turned on.